exercises/matrix/canonical-data.json

```diff
@@ -0,0 +1,52 @@
+{
+  "exercise": "matrix",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "one number matrix",
+      "property": "rows",
```

Comment:
Seeing how only a single index is being used, shouldn't this be named `row` (singular)?